llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -628,13 +628,13 @@ bool CompileTimePropertiesPass::transformSYCLPropertiesAnnotation(
   // Read the annotation values and create the new annotation string.
   std::string NewAnnotString = "";
   auto Properties = parseSYCLPropertiesString(M, IntrInst);
-  for (auto &Property : Properties) {
+  for (const auto &[propName, propVal] : Properties) {
```

Comment:
Can we use capital initial letters `PropName` `PropVal`, that's more standard in the codebase

Reply:
[c796b9e](https://github.com/intel/llvm/pull/11458/commits/c796b9e063313c04dcda28b0feeb46f08b0fbcff)